src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -283,6 +294,23 @@ private HearingEntity saveHearingDetails(HearingRequest createHearingRequest) {
         return hearingRepository.save(hearingEntity);
     }
 
+    private void savePartyRelationshipDetails(HearingRequest createHearingRequest, HearingEntity hearingEntity) {
+
+        final List<PartyDetails> partyDetails = createHearingRequest.getPartyDetails();
+        final List<HearingPartyEntity> hearingParties = hearingEntity.getLatestCaseHearingRequest().getHearingParties();
+
+        if (partyDetails != null) {
+            for (PartyDetails partyDetail : partyDetails) {
+                if (partyDetail.getIndividualDetails() != null
+                    && partyDetail.getIndividualDetails().getRelatedParties() != null) {
```

Comment:
```suggestion
                    && !Collections.isEmpty(partyDetail.getIndividualDetails().getRelatedParties()) {
```

Reply:
Fixed in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/768e22b379256ea506790b5fc818777349748c05